lsql/judge/models.py

```diff
@@ -173,6 +173,26 @@ def num_submissions_by_user(self, user):
         """Number of user submissions to the problem"""
         return Submission.objects.filter(problem=self, user=user).count()
 
+    def solved_n_position(self, position):
+        """Name of the user who solved the problem in 'position' position"""
```

Comment:
Dos últimos detalles por dejarlo perfecto: estas funciones no devuelve el nombre del usuario sino que devuelven un usuario (objeto User). Actualiza el comentario para evitar confusiones.

Reply:
`"""User who solved the problem in 'position' position"""` ?

Reply:
Por ejemplo, sí. Y similar en las otras tres.